content/en/docs/reference/command-line-tools-reference/feature-gates/ComponentSLIs.md

```diff
@@ -13,6 +13,12 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.27"
+    toVersion: "1.31"
+  - stage: stable
+    locked: true
```

Comment:
```suggestion
```
as #49388 isn't implemented yet.

Reply:
Does it hurt?
Do you mean we revisit this features again to find if they are locked or not later?

Reply:
I think there is a cost here; people might see this field and infer it's implemented. Finding out that it's not can be a surprise, replicated across multiple contributors.

I can see a small chance of a painful surprise, where maybe someone relies on this value and they shouldn't have.

Reply:
It is not about you think. People read the rendered HTML, not the YAML.
I'm all for removing this information if you sign up to add this back.

Reply:
Noted; I still have the qualm I had before.

Reply:
@tengqm would you be willing to start a Slack discussion about the way forward?